deployment/terraform/azure/container_app/examples/azure-integration/main.tf

```diff
@@ -0,0 +1,107 @@
+variable "port_client_id" {
```

Comment:
I would suggest moving the variables to a separate variables.tf file
If you want to keep everything in one file for the example, also put the providers.tf content here - I would recommend to split them up

Reply:
splitted